metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -95,7 +95,7 @@ class BuildServerConnection private (
       "1.4.10",
       version,
     )
-    isSbt || (isBloop && supportNewDebugAdapter)
+    isSbt || isScalaCLI || (isBloop && supportNewDebugAdapter)
```

Comment:
We're already doing this so it's not a huge deal, but I thinking out loud, I wonder if there's a better way to do this instead of manually adding these. I wonder if using something like `data` in the `InitializeBuildResult` from the server could just have key there saying `debugAdapterVersion` or something. Then we could check this dynamically instead of relying on this here.